pages/automatedtestingpython.md

```diff
@@ -72,10 +72,19 @@ Finally:
 
 ## MacOS/Ubuntu - Installing Testing Development Environment
 ### Installing Nation
-WIP - Needs some love.
+The process is similar to installing Nation for Windows.
 
 ### Installing Python, Pip, Selenium, Geckodriver, and Firefox
-WIP - Needs some love.
+1. Install Python 3 with `brew install python3` or download Python from https://www.python.org/downloads/release/python-352/)
```

Comment:
ooh, AFAIK, `brew` is specific to mac? we need to add some information about installing python in ubuntu, let say via yum and apt?

If you don't have linux try free VM here https://c9.io